.travis.yml

```diff
@@ -9,6 +9,24 @@ sudo: false
 services:
     - postgresql
 
+addons:
+  postgresql: "13"
+  apt:
+    sources:
+      - sourceline: deb http://apt.postgresql.org/pub/repos/apt/ jammy-pgdg main
```

Comment:
if you have some time to spare, it would be nice to replace this annoying setup with GH actions following @Avantol13's doc [here](https://docs.google.com/document/d/1wf1hQK0hQsVzaK23fIiktrnxYdSgRxOag2VSzUBHFhc) and example [here](https://github.com/uc-cdis/sheepdog/pull/400)

Reply:
Yeah let me knock that out real quick.